deploy/dep.yaml

```diff
@@ -77,12 +83,13 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
-        - name: SECRET_CONFIG_PATH
-          value: /etc/storage_ibmc
         volumeMounts:
-          - mountPath: /etc/storage_ibmc
-            name: customer-auth
+          - mountPath: /var/run/secrets/tokens
+            name: vault-token
       volumes:
-        - name: customer-auth
-          secret:
-            secretName: storage-secret-store
+        - name: vault-token
+          projected:
+            sources:
+            - serviceAccountToken:
+                path: vault-token
+                expirationSeconds: 600
```

Comment:
We also might need some [manifest changes here](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/blob/b0497287b92f6e545445dfd545615178089a25d9/assets/node.yaml#L39-L45).
I know we create storage-secret-store today, don't know where vault-token comes from, but it looks like at least the expected mount path changed?

Reply:
This is part to a managed token, something like STS on AWS. We don't use that auth on IBM (yet).